prep_ubuntu.sh

```diff
@@ -51,8 +51,13 @@ upgrade_base_operating_system() {
   # now install it
   apt-get -y -q --no-install-recommends --force-yes install unzip iptables iptables-persistent
   systemctl enable netfilter-persistent
+
+  # allow root logon to SSH
```

Comment:
This line has "gotten lost". It belongs in the earlier SSH/logon commit

Reply:
Fixed.